lucene/sandbox/src/java/org/apache/lucene/sandbox/rabitq/Search.java

```diff
@@ -33,16 +36,17 @@ public static void main(String[] args) throws Exception {
     String dataset = args[1]; // eg "gist"
     int numCentroids = Integer.parseInt(args[2]);
     int dimensions = Integer.parseInt(args[3]);
-    int B_QUERY = Integer.parseInt(args[4]);
+    int B_QUERY = 4;
```

Comment:
lets just delete `B_QUERY` and this also means all the `args` > 3 will dec by 1.